Add "buttonsStyle" and "buttonsLayout" options

pjanik opened the pull request:
You can switch between icons and text and also decide whether buttons should be on the right side or below the title. I've merged this branch into gh-pages, so you can play with new options here:
http://concord-consortium.github.io/lab-grapher/examples/#earth-surface-temperature

I know this code adds even more fiddling with magic values of paddings, font sizes etc., but that's general grapher's style and I don't think it makes sense to fight with it right now. I would really consider rewriting grapher at some point, I believe that motivated developer can do it in 2-4 weeks. :wink:

scytacki commented:
What should the 'useIcons' checkbox do here: http://concord-consortium.github.io/lab-grapher/examples/#earth-surface-temperature ?

pjanik commented:
@scytacki, ah, I should have removed it, it was previous version of 'buttonsStyle' and I committed it by accident.

pjanik commented:
Okay, I'm assuming that it was the only comment which is now addressed, so I'm merging this PR. If there is another feedback, just let me know here and I'll work on that too.

pjanik commented:
When I started testing these options in Lab, I noticed one issue which is now fixed by: 1567189ec5f581597cd501e9de29fb54f41f5288
I've also added 'Font Size' checkbox to the example page. It changes font size of the graph container and calls `graph.resize()`. It's exactly what happens in Lab, so it's useful to play with different values if you're working on grapher layout.
